lib/perfBaseTest.py

```diff
@@ -38,30 +38,20 @@ def setUp(self):
         # wait browser ready
         self.get_browser_done()
 
-        # execute pre-run-script.
-        # You have to specify the pre_run_script and test_url before calling parent setup in your test class
-        if os.getenv("PRE_SCRIPT_PATH"):
-            pre_script_args = []
-
-            # clone pre run script test url id
-            if hasattr(self, "pre_script_args"):
-                pre_script_args = self.args_parser(os.getenv("PRE_SCRIPT_PATH"), self.pre_script_args)
-
-            # execute pre run script
-            self.round_status = self.sikuli.run_test(os.getenv("PRE_SCRIPT_PATH"),
-                                                     os.getenv("PRE_SCRIPT_PATH") + "_" + self.env.time_stamp,
-                                                     args_list=pre_script_args)
-
         # capture 1st snapshot
         time.sleep(5)
 
+        if self.index_config['snapshot-timing'] == self.global_config['default-snapshot-timing-base']:
+            videoHelper.capture_screen(self.env, self.env.video_output_sample_1_fp, self.env.img_sample_dp,
+                                       self.env.img_output_sample_1_fn)
+
         if self.env.PROFILER_FLAG_AVCONV in self.env.firefox_settings_extensions:
             if self.env.firefox_settings_extensions[self.env.PROFILER_FLAG_AVCONV]['enable'] is True:
                 videoHelper.capture_screen(self.env, self.env.video_output_sample_1_fp, self.env.img_sample_dp,
                                            self.env.img_output_sample_1_fn)
         time.sleep(2)
 
-        # Record timestamp t1
+        # Record timestamp t2
```

Comment:
this should be t1. or, do you want "initial timestamp -> t1" and "t2->t3"?
might be a typo since there is no change for webdriverBaseTest